libc/include/llvm-libc-macros/float-macros.h

```diff
@@ -9,163 +9,206 @@
 #ifndef LLVM_LIBC_MACROS_FLOAT_MACROS_H
 #define LLVM_LIBC_MACROS_FLOAT_MACROS_H
 
-// Suppress `#include_next is a language extension` warnings.
-#ifdef __clang__
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wgnu-include-next"
-#pragma clang diagnostic ignored "-Winclude-next-absolute-path"
-#else // gcc
-#pragma GCC system_header
-#endif //__clang__
-
-#include_next <float.h>
-
-#ifdef __clang__
-#pragma clang diagnostic pop
-#endif //__clang__
-
-#ifndef FLT_RADIX
+#ifdef __FLT_RADIX__
 #define FLT_RADIX __FLT_RADIX__
+#else 
+#define FLT_RADIX 2
 #endif // FLT_RADIX
 
-#ifndef FLT_EVAL_METHOD
+#ifdef FLT_EVAL_METHOD
```

Comment:
This should be `#ifdef __FLT_EVAL_METHOD__`.  Update similar checks in this file.